mahta/JS/JS basix syntax/script.js

```diff
@@ -0,0 +1,88 @@
+function add(a,b){
+    return(a + b);
+}
+
+function flipACoin() {
+    var random = Math.random();
+    if (random > 0.5) {
+        retuen("tail");
+    } else {
+        return("head");
+    }
+}
+
+function concat(stringA, stringB) {
+    return(stringA.concat(stringA, stringB));
+}
+
+function power2(x) {
+    return(2**x);
+}
+
+function getUser(firstName, lastName) {
+    const FirstName=firstName.replace(firstName.charAt(0),firstName.charAt(0).toUpperCase());
```

Comment:
A best practice is to use camelCase for these kinds of variables, not snake_case or PascalCase or etc..